ppml/tdx/docker/trusted-bigdl-llm-serving-tdx/Dockerfile

```diff
@@ -1,46 +1,22 @@
-FROM ubuntu:20.04
+FROM intelanalytics/bigdl-llm-serving-cpu:2.4.0-SNAPSHOT
 
 ARG http_proxy
 ARG https_proxy
-ARG TINI_VERSION=v0.18.0
 
 RUN mkdir -p /ppml/
 
 
 ADD ./entrypoint.sh /opt/entrypoint.sh
-ADD ./attestation_cli.py /opt/attestation_cli.py
 ADD ./attestation.sh /opt/attestation.sh
-ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /sbin/tini
 
 
-# Install PYTHON 3.9
-RUN env DEBIAN_FRONTEND=noninteractive apt-get update && \
-    apt install software-properties-common libunwind8-dev vim less -y && \
-    add-apt-repository ppa:deadsnakes/ppa -y && \
-    apt-get install -y python3.9 git curl wget && \
-    rm /usr/bin/python3 && \
-    ln -s /usr/bin/python3.9 /usr/bin/python3 && \
-    ln -s /usr/bin/python3 /usr/bin/python && \
-    apt-get install -y python3-pip python3.9-dev python3-wheel && \
-    pip3 install --no-cache --upgrade requests argparse cryptography==3.3.2 urllib3 && \
-    pip3 install --pre --upgrade bigdl-llm[all] && \
-    cd /ppml/ && \
-    curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && \
-    # Install FastChat from source requires PEP 660 support
-    python3 get-pip.py && \
-    rm get-pip.py && \
-    # Install FastChat
-    git clone https://github.com/analytics-zoo/FastChat.git && \
-    cd /ppml/FastChat && \
-    git checkout dev-2023-08-01 && \
-    pip install -e . && \
-    # Pin gradio version because this error:https://github.com/lm-sys/FastChat/issues/1925
-    pip install --pre --upgrade gradio==3.36.1 && \
-    pip install --pre --upgrade bigdl-nano && \
+# Change workdir
+RUN cp -r /llm/FastChat /ppml/FastChat && \
```

Comment:
I am not sure whether this would works...

Can you run `pip list | grep fschat` after this step and verifies that it points to the right position?